plugin/completion.py

```diff
@@ -243,6 +243,24 @@ def format_completion(self, item: dict) -> 'Tuple[str, str]':
                 hint = completion_item_kind_names[kind]
         # label is an alternative for insertText if insertText not provided
         insert_text = item.get("insertText") or label
+        # try to handle textEdit if present
+        text_edit = item.get("textEdit")
+        if text_edit:
```

Comment:
Can we hide this behaviour behind a global LSP setting for now? I'd like to avoid hitting this code path automatically for all language servers that happen to send textEdit.